plugins/lexon/profile.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "lexon",
+    "displayName": "Lexon",
+    "version": "0.1.1-alpha",
+    "methods": ["getCompilationResult", "compile"],
```

Comment:
is this plugin already implement these functions?
https://github.com/ethereum/remix-plugins-directory/blob/master/common-api/compiler/api.ts#L12

Reply:
It somewhat does, but probably not as it should and they are not exported. Tbh, I just copied the vyper entry and thought these were necessary. It does implement compile, but just internally with different syntax. Could we merge it before I implement them properly?